agent-packaging/Dockerfile

```diff
@@ -22,4 +22,4 @@ COPY --from=0 /daisy daisy
 
 ENV GOOGLE_APPLICATION_CREDENTIALS /etc/compute-image-tools-test-service-account
 
-ENTRYPOINT ["/bin/bash", "-c", "./main.sh"]
+ENTRYPOINT ["/bin/bash", "-c", "/run_daisy_workflow.sh"]
```

Comment:
is this coming from line 20 (`COPY packaging-scripts /`) ?

i thought we decide to move this to guest-test-infra repo? otherwise this will have to be built by hand

Reply:
main.sh was just used for testing purpose.

as discussed earlier today, let's keep everything in osconfig repo for now. I will move the packaging docker image and some refactored scripts to guest-test-infra repo